orttraining/orttraining/python/training/optim/_ds_modifier.py

```diff
@@ -39,14 +39,22 @@ def can_be_modified(self):
         # it's safe to update the version supporting list. Otherwise, or the file is moved or renamed,
         # we need to check the implementation of these functions in detail.
         ds_version = Version(deepspeed.__version__)
-        if ds_version > Version("0.7.3") or ds_version < Version("0.4.0"):
+        if ds_version > Version("0.8.0") or ds_version < Version("0.4.0"):
             warnings.warn(
                 "Skip modifying optimizer because of unsupported DeepSpeed version {}, "
                 "supported version: 0.4.0 - 0.7.3.".format(deepspeed.__version__),
```

Comment:
Do you need to update this to 0.8.0? Can we create a variable with version number for start and end versions and reference that instead of making changes everywhere?

Reply:
there's only place needs the update, thus there's no need to create start/end variables

Reply:
The warning message has 0.7.3 in it. I think we missed updating that.

Reply:
```suggestion
                "supported version: 0.4.0 - 0.8.0.".format(deepspeed.__version__),
```